test/e2e/sagemaker/resources/endpoint.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Endpoint
+metadata:
+  name: $ENDPOINT_NAME
+spec:
+  endpointName: $ENDPOINT_NAME
+  endpointConfigName: $ENDPOINT_NAME-config
+  tags:
+    - key: var
+      value: val
```

Comment:
Unless we have explicit tests for tags (which I don't see), these seem pointless. 
Same with all other configuration files